pipeline/scraper/github_query.py

```diff
@@ -92,24 +98,27 @@ def run_query(today, language='JavaScript'):
 
     # fetch data and write to database
     last_node = None
-    monthly_search_str = get_monthly_search_str(today)
```

Comment:
Do we need `get_monthly_search_str()` and related code anymore? Just remove that whole file?